acme/certificate_challenges.go

```diff
@@ -15,26 +14,41 @@ import (
 	"github.com/go-acme/lego/v4/providers/http/webroot"
 	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vancluever/terraform-provider-acme/v2/acme/dnsplugin"
 )
 
-func setCertificateChallengeProviders(client *lego.Client, d *schema.ResourceData) error {
+// setCertificateChallengeProviders sets all of the challenge providers in the
+// cilent that are needed for obtaining the certificate.
```

Comment:
cilent -> client

Reply:
Thanks for the catch!

Reply:
Thanks for the DNS fix! I've been debugging this for days on GKE. Just deployed to dev w/ fingers crossed issue resolved.

Reply:
Fingers crossed and let me know if it works for you!